core/src/main/java/org/kiwix/kiwixmobile/core/utils/DimenUtils.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.utils
+
+import android.app.Activity
+import android.content.Context
+import android.util.DisplayMetrics
+import android.util.TypedValue
+import androidx.appcompat.R.attr
+
+/**
+ * Created by hritik on 03/23/20.
+ */
+object DimenUtils {
+  @JvmStatic fun getToolbarHeight(context: Context): Int {
+    val t = TypedValue()
+    context.theme.resolveAttribute(attr.actionBarSize, t, true)
+    return context.resources.getDimensionPixelSize(t.resourceId)
+  }
+
+  @JvmStatic fun getWindowHeight(activity: Activity): Int {
+    val displayMetrics = DisplayMetrics()
+    activity.windowManager.defaultDisplay.getMetrics(displayMetrics)
+    return displayMetrics.heightPixels
+  }
+
+  @JvmStatic fun getWindowWidth(activity: Activity): Int {
```

Comment:
All of these functions should be changed to extension functions. This will help a lot when everything is changed to kotlin as nobody is going to revisit this file to change them to extension functions later.

Reply:
@akkywadhwa this comment should be addressed

Reply:
@macgills I'll surely do.

Reply:
Changing these functions to extension functions will require a change in the following files: 

1. TabsAdapter.java
2. CoreMainActivity.java
3. ToolbarScrollingKiwixWebView.java
4. ToolbarStaticKiwixWebView.java
 
Should I change to extension functions or leave it for later?

PS - #1961 requires the same changes. I will update it once this is finalized.
Tags: @abdulwd and @macgills

Reply:
Changing them to extension functions should incur no change from java calling code, see this [PR](https://github.com/kiwix/kiwix-android/pull/2000/files) where I didn't change a line of java